CODEOWNERS

```diff
@@ -59,6 +59,7 @@ Kconfig*                                  @tejlmand
 /lib/at_notif/                            @pkchan
 /lib/agps/                                @jtguggedal @simensrostad
 /lib/nrf_modem_lib/                       @rlubos
+/lib/edge_impulse/                        @pdunaj
```

Comment:
Nothing wrong here, but cannot place comments in the commit message.

Could you provide a bit more info in the commit message.
I already had a suggestion in here: https://github.com/MarekPieta/fw-nrfconnect-nrf/pull/7
```
Using ExternalProject for building the edge impulse library.

This allows downloading and extracting the zip file containing the model
on each build invocation.

In the event that sources are added or removed from the library the
ExternalProject ensures that the build system will correctly handle this
in the build system.
```

Reply:
Ok, added